eacl/table.go

```diff
@@ -199,3 +199,25 @@ func (t *Table) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// EqualTables compares Table with each other.
+func EqualTables(t1, t2 Table) bool {
+	if !t1.CID().Equal(t2.CID()) ||
+		t1.Version() != t2.Version() {
```

Comment:
`!=` is working but not safe, I'd define equality function for `version.Version` type.